app/components/article-content.js

```diff
@@ -388,6 +390,36 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Handles opening sections when click event occurs on reference
+		 *
+		 * @returns {void}
+		 */
+		handleReferences(event) {
+			if (event.target.closest('a[href^="#cite_ref-"]')) {
+				const element = event.target.closest('a[href^="#cite_ref-"]'),
```

Comment:
I know we are still not consitent with that 
but per our current js style guide- https://github.com/Wikia/javascript-style-guide#variables
one const per variable

we need to finally work on our eslinter...

Reply:
also you get this element twice, could be optimized to get it just once maybe?